modules/tinymce/src/themes/silver/main/ts/backstage/StyleFormatsBackstage.ts

```diff
@@ -17,7 +18,7 @@ export const init = (editor: Editor) => {
   const getPreviewFor: FormatRegister.GetPreviewForType = (format) => () => {
     const fmt = editor.formatter.get(format);
     return fmt !== undefined ? Optional.some({
```

Comment:
Could we do `Optional.from(editor.formatter.get(format)).map(...)`?

Reply:
I guess this isn't changed code so don't worry about it

Reply:
Outside of the scope of this JIRA. Please remember that for types we don't want to change functionality if possible.